Sources/AsyncSequenceValidation/TaskDriver.swift

```diff
@@ -28,11 +30,16 @@ func start_thread(_ raw: UnsafeMutableRawPointer) -> UnsafeMutableRawPointer? {
   Unmanaged<TaskDriver>.fromOpaque(raw).takeRetainedValue().run()
   return nil
 }
-#elseif canImport(Glibc) || canImport(Musl)
+#elseif (canImport(Glibc) && !os(Android)) || canImport(Musl)
```

Comment:
These two changes get `start_thread()` working with Swift 5.10 and NDK 26 also, by changing the signature for the nullability annotations. That's why this is the only portion of this pull to check `os(Android)`, because that's available in both Swift 5.10 and 6.